node-eclib.js

```diff
@@ -191,17 +191,24 @@ ECLib.prototype = {
         // the 6th fragment.
         var done = 0;
 
-        fragmentIds.sort();
+        fragmentIds.sort(function(a, b) {
+            return (a - b);
+        });
+        var len = fragmentIds.length;
+        var recFrags = new Array(len);
 
         // Recover all missing fragments one by one.
-        fragmentIds.forEach(function reconstructEach(id) {
+        fragmentIds.forEach(function reconstructEach(id, index) {
```

Comment:
Additionally, if you use an arrow function here you don't have to bind this for context.